OsmAnd-java/src/main/java/net/osmand/gpx/GPXTrackAnalysis.java

```diff
@@ -47,6 +47,21 @@ public class GPXTrackAnalysis {
 	public float maxSpeed = 0;
 	public float avgSpeed;
 
+	public float maxSensorSpeed = 0;
```

Comment:
These all are [GpxParameter.java](https://github.com/osmandapp/OsmAnd/pull/18735/files#diff-6c1c1cc00c7e408761a96c8ed400ac7e97bf2277fbfc2b92057cc60ff7c21819)

Reply:
And it's important even to check which parameter analysis support